### ISSUE TITLE
SVD-assist if model runs fail during Jacobian calculation

### ISSUE THREAD

**Issue by Paszka1** [pypest/pyemu#439]
I am performing an SVD-assisted inversion with `pestpp-glm` using both super and base parameter iterations. Some of the runs fail during super-parameter iterations and as a result pestpp-glm will switch to base parameter iteration `at the end` of the super-parameter iteration as it is unable to calculate the Jacobian. It seems to me that it is a waste of time to wait until the super-parameter iteration finishes, isn't it? Is there a specific reason for that? Please get me enlightened.

**Comment by jtwhite79** [comment]
pestpp-glm breaks out of the super par iter if any super par jco runs fail - it does this after trying to make the jco runs - until the run Mgr returns control, pestpp-glm has no way of knowing that any runs failed...

**Comment by Paszka1** [comment]
Thanks Jeremy,

Sounds logical, but it's still confusing as the Mgr. reports failed runs during the iteration, so one would expect `pestpp-glm` could also be informed on the fly. I learned the lesson now, and next time when I have difficult runs I'll pay attention not to use svd-assist.

On another note. I added a discussion topic about a month ago that apparently wasn't yet noticed. I would appreciate if you guys take a look at it and give me some feedback at your earliest convenience.
